mitiq/mitiq_qiskit/conversions.py

```diff
@@ -60,6 +76,7 @@ def from_qiskit(circuit: QuantumCircuit) -> cirq.Circuit:
     Returns:
         Mitiq circuit representation equivalent to the input Qiskit circuit.
     """
+    circuit = _remove_barriers(circuit)
```

Comment:
If we decide to keep  `_remove_barriers` in the body of `from_qasm`, maybe we can remove  `_remove_barriers` from the body of `from_qiskit` (since `from_qasm` is called one line below).
```suggestion
```